README.md

```diff
@@ -19,14 +17,28 @@ Dependencies:
 * Virtualbox
 * virtualization enabled in bios
 
-On Linux systems:
+Procedure:
+
+1. Clone the pelux-manifests git repository.
+2. Start vagrant
 ```bash
-MANIFEST=<manifest> BRANCH=<branch> vagrant up
+MANIFEST=<manifest> vagrant up
 ```
 
+Note:
+The virtual machine started via vagrant will sync the cloned git repository and use the manifests contained in it
```

Comment:
Since there are multiple manifests to choose from in the `pelux-manifests` repo i think it is fair to say manifests. I guess only one is pointed to but they are linked and if you point to `pelux-intel-qt.xml` that would import all of the other manifests as well.